### DISCUSSION TITLE
error when upgrading

### DISCUSSION THREAD

**Discussion by constantius24** [mocaccinoOS/mocaccino#132] [General]
Error: failed computing upgrade: Failed solving solution for upgrade: Could not compute upgrade - couldn't uninstall candidates : could not satisfy the constraints: 
          libglade-libs-2.6.4-r3+4 and 
          gtk2-libs-2.24.33-r3+3 and 
          !(libglade-libs-2.6.4-r3+4) or gtk2-libs-2.24.33-r3+2 and 
          !(gtk2-libs-2.24.33-r3+2) or !(gtk2-libs-2.24.33-r3+2) or !(gtk2-libs-2.24.33-r3+3)

**Comment by msdobrescu** [comment]
This is a common behavior.
Try reinstalling the mentioned packages:
`luet reinstall libs/libglade libs/gtk2`

**Comment by constantius24** [comment]
hi. Reinstalled but nothing help. Removeing cairo-clock and then works. But i have 2 computers with MocaccinoOS. ONE with Gnome and one with KDE. Which one with Gnome no problem. Problem occurs with KDE

**Comment by msdobrescu** [comment]
It depends on the deps of the installed packages, of what else is installed.
During a full upgrade, the same packages may repeat. It stabilizes in the end.
I am a KDE user, but I don't use cairo dock.

At the end of upgrades, it may be useful to run `luet oscheck --reinstall` in case some file is missing.

If you think it's a luet bug, add a report [here](https://github.com/mudler/luet).
